share/doc/revng/user-manual/analyses.md

```diff
@@ -50,7 +50,7 @@ $ revng artifact \
         | revng ptml | grep -A2 -B1 '\bmain\b'
 _ABI(SystemV_x86_64)
 generic64_t main(generic64_t _argument0) {
-  return _argument0 * 3 & 0xFFFFFFFF;
+  return _argument0 * 3 & 4294967295;
```

Comment:
This does not sound right 🤔

Reply:
This doesn't look right to me either. But it's being addressed in revng-c. It should go away soon.

Reply:
Solved! The issue was that the `pretty-int-formatting` was not handling the implicit casts.